### PR TITLE
Fix/readme 391

### PR DESCRIPTION
**Pull request by sateeshperi**

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -23,21 +23,26 @@ The second workflow uses [BWA-Meth](https://github.com/brentp/bwa-meth) and [Met
...
 ```csv title="samplesheet.csv"
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L003_R1_001.fastq.gz,AEG588A1_S1_L003_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L004_R1_001.fastq.gz,AEG588A1_S1_L004_R2_001.fastq.gz
+sample,fastq_1,fastq_2,genome
+SRR389222_sub1,SRR389222_sub1.fastq.gz,,
+SRR389222_sub2,SRR389222_sub2.fastq.gz,,
+SRR389222_sub3,SRR389222_sub3.fastq.gz,,
+Ecoli,Ecoli_10K_methylated_R1.fastq.gz,Ecoli_10K_methylated_R2.fastq.gz,
```
Comment (maxulysse) [inline docs/usage.md:45]: is `Ecoli` the sample or the genome?

Comment (maxulysse) [reply docs/usage.md:45]: shouldn't `Ecoli_10K_methylated` be the sample?

Comment (sateeshperi) [reply docs/usage.md:45]: I was trying to just use the one in the test samplesheet. 

I can change it in the samplesheet but, will need to update all tests and snapshots

Comment (sateeshperi) [reply docs/usage.md:45]: insert interstellar meme - this one small step is going to cost us 7 years!!!! 🚀

Comment (sateeshperi) [reply docs/usage.md:45]: on it

Comment (sateeshperi) [reply docs/usage.md]: should be fixed now @maxulysse

**Review by maxulysse — APPROVED** [review]

File: docs/usage.md (hunk 1 above)
Comment (maxulysse) [reply docs/usage.md]: yeah sorry about that, but for me `Ecoli` could be actually a good usage of this new genome field.

**Pull request merged**